src/Tizen.Security/Tizen.Security/Privilege.cs

```diff
@@ -173,7 +177,7 @@ public static string GetPrivacyDisplayName(string privilege)
         /// <remarks>The privilege must be privacy related.</remarks>
         /// <returns>Status true if the privilege is on and false if the privilege is off.</returns>
         /// <exception cref="System.ArgumentNullException">Thrown when there is a null parameter.</exception>
-        /// <exception cref="System.ArgumentException">Thrown when there is an invalid parameter.</exception>
+        /// <exception cref="System.ArgumentException">Thrown when there is an invalid parameter such as a non-existing privilege.</exception>
         /// <exception cref="System.OutOfMemoryException">Thrown when out of memory occurs.</exception>
         /// <exception cref="System.InvalidOperationException">Thrown when an internal error occurs.</exception>
         /// <exception cref="System.NotSupportedException">The required feature is not supported.</exception>
```

Comment:
This issue is not related to your pull request, but you are changing a comment in the API item which is deprecated since API5. Current API level is API8. Please check our deprecation policy, because maybe this API item should be removed instead or hidden by using EditorBrowsableAttribute.

Reply:
Revert the changes. I'm preparing another ACR for removal of this API.